mkosi

```diff
@@ -2864,16 +2864,18 @@ def install_unified_kernel(args: CommandLineArguments,
     if do_run_build_script:
         return
 
-    if args.distribution not in (Distribution.fedora, Distribution.mageia):
+    if args.distribution not in (Distribution.fedora, Distribution.mageia, Distribution.ubuntu, Distribution.debian):
         return
+    
+    lib_path = 'lib' if args.distribution in (Distribution.ubuntu, Distribution.debian) else 'usr/lib'
```

Comment:
On an unrelated note, is this strictly necessary? Current versions of Debian default to `merged-usr` and although Debian probably won't ever mandate this, I guess we can be opinionated here. At least on my machine I see `lib -> usr/lib` when doing `ls -l /`.

Reply:
For Debian, this is actually not needed since it has that sym link, on Ubuntu though it's needed.

Reply:
I quickly booted an Ubuntu 19.04 live usb system, that I had flying around, so this may only count as anecdotal evidence, but it does have the `lib -> usr/lib` symlink as well. I also googled around a bit and it seems merged-usr is the default since 19.04.

Reply:
Yes, this is right. I am doing this on Ubuntu 18.04 though, but would you suggest instead to create a symlink in my setup and just change this PR to enable the Ubuntu flag?

Reply:
Good question. Looking at `install_debian_or_ubuntu` debootstrap would already install the system as merged usr if it knew it. I guess you could convert the system to merged usr afterwards by installing the `usrmerge` package, that should be available in 18.04 as well (haven't checked that though). I don't really know what the best way forward is, but specialcasing Ubuntu for a two-year old LTS when the new one is around the corner doesn't seem optimal to me.

Reply:
Yeah, you are right, it's best to keep it simple. I will make changes based on this.